app/main/views/suppliers.py

```diff
@@ -82,6 +82,9 @@ def dashboard():
             )
         })
 
+    if "currently_applying_to" in session:
```

Comment:
Why remove this here?

Reply:
If the supplier is on the account dashboard and goes to their company details, they aren't coming "from" an application - so they shouldn't get a link on the page which says "go back to the application".